apps/bot/src/util/Constants.js

```diff
@@ -1,9 +1,13 @@
-const EMBED_COLORS = { default: 0x6FA8DC }
+/**
+ * @readonly
+ * @enum {number}
+ */
+const EmbedColors = { DEFAULT: 0x6FA8DC }
```

Comment:
change constant name to uppercase. The project standard has not yet been defined, but it will probably be like this